src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -686,12 +643,14 @@ Amplify.API.mutate(ModelMutation.update(cartWithRemovedCustomer),
 
 <InlineFilter filters={["swift"]}>
 
+You can set the relationship field to `nil` to delete a "Has One" relationship between records.
+
 ```swift
-let cartWithRemovedCustomer = Cart(id: existingCart.id, items: existingCart.items, customer: nil)
-let cartRemoveRequest = try await Amplify.API.mutate(request: .update(cartWithRemovedCustomer))
-guard case .success = cartRemoveRequest else {
-    print("API response: \(cartRemoveRequest)")
-    return
+do {
+    existingCart.setCustomer(nil)
+    let cartWithCustomerRemoved = try await Amplify.API.mutate(request: .update(existingCart)).get()
+} catch {
+    print("Failed to remove customer from cart", error)
 }
```

Comment:
<img width="1017" alt="image" src="https://github.com/aws-amplify/docs/assets/1365977/c49223ee-9e19-4af5-aa98-087fab4d0faf">

- setting to `nil` on existing cart instead of creating a new instance